plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/LocalCheInfrastructureProvisioner.java

```diff
@@ -89,7 +94,26 @@ public void provision(Environment envConfig, CheServicesEnvironmentImpl internal
                    .add(SystemInfo.isWindows() ? pathEscaper.escapePath(projectFolderVolume)
                                                : projectFolderVolume);
 
+        if (System.getenv(CHE_USER_ID) != null) {
+            addVolume(internalEnv, devMachineName, ETC_PASSWD_VOLUME);
```

Comment:
Please add unit tests to cover changed code. Unfortunately there are no tests yet for this class. But you can cover only your code if you will.